### PR TITLE
lock network until all nodes are registered in fault tolerance test

### PR DESCRIPTION
**Pull request by fabioDMFerreira**
## Fixes

* Fault tolerance test `Sharder goes down for 3 minutes simultaneously with coming up of the previous one`

## Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

## Associated PRs (Link as appropriate):
- blobber:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: code/go/0chain.net/conductor/conductor/executor.go [hunk 1]
```diff
@@ -301,6 +301,11 @@ func (r *Runner) WaitShareSignsOrShares(ssos config.WaitShareSignsOrShares,
 }
 
 func (r *Runner) WaitAdd(wadd config.WaitAdd, tm time.Duration) (err error) {
+	// filter initialized nodes
+	// nodes := r.server.Nodes()
+	// for name := range nodes {
+	// 	wadd.Take(name)
+	// }
```
Comment (peterlimg) [inline code/go/0chain.net/conductor/conductor/executor.go]: Remove commented code if not used

**Review by peterlimg — CHANGES_REQUESTED** [review]
Please merge staging code to avoid sharders load snapshot panic during system tests

File: code/go/0chain.net/conductor/conductor/executor.go (hunk 1 above)
Comment (fabioDMFerreira) [reply code/go/0chain.net/conductor/conductor/executor.go]: fixed in [dd64987](https://github.com/0chain/0chain/pull/1851/commits/dd64987f770e7daad5607ab8a72413cc69a5b5bb)

**Review by peterlimg — APPROVED** [review]
LGTM

**Comment by fabioDMFerreira** [comment]
@peterlimg the system tests are failing. I don't think the changes have any relation to the failure. Could you merge this PR?

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | TRUE  |
| system-tests  | master  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | fix/fault-tolerance-tests-network-locking  |
| blobber  | staging  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | master  |
| 0proxy  | staging  |
| 0search  | staging  |

**Review by peterlimg — APPROVED** [review]

**Pull request merged**